docs/utils/nft-borsh.ts

```diff
@@ -53,7 +53,7 @@ export const NFTOKEN_MINTLIST_CREATE_IX = new FixableGlowBorsh<{
   metadata_url: string;
   collection_metadata_url: string;
   go_live_date: DateTime;
-  price: Solana.SolAmount;
+  price_lamports: Solana.SolAmount;
```

Comment:
I don't think we should make this change. `price_lamports` would be type `string`